packages/plugins/ui-theme/src/i18n/crowdin/ui.json

```diff
@@ -180,8 +180,9 @@
     "chineseTraditional": "Chinese Traditional"
   },
   "flag": {
-    "austria": "Austria",
```

Comment:
😢 could you put Austria back? My friend @priscilawebdev created the UI from Austria 🇦🇹

Reply:
There was no Austrian flag before (just Australia), so I thought it was a typo. I will of course add Austria!